src/components/cart/cart-item/cart-item.module.scss

```diff
@@ -95,3 +114,36 @@
     width: 20px;
     height: 20px;
 }
+
+/* Cart item styles used in right panel */
+.smallCartItem .productInfo {
+    grid-template-columns: 3fr 3fr 1fr;
+    grid-template-areas:
+        'productNameAndPrice productNameAndPrice removeButton'
+        'quantity             priceBreakdown   priceBreakdown';
+}
+
+.smallCartItem .quantity {
+    justify-self: start;
+}
+
+.smallCartItem .priceBreakdown {
+    justify-self: end;
+}
+
+@media (max-width: $tablet-width) {
```

Comment:
You have `.smallCartItem` but then you also have styling for `width < tablet-width`. I don't understand how these are supposed to interact.

Reply:
Ah, they're not supposed to interact. They're duplicated.